README.md

```diff
@@ -152,11 +142,47 @@ INFO: seed corpus: files: 755 min: 1b max: 35982b total: 252654b rss: 35Mb
 
 The crash is saved under `/tmp/fuzzing/artifacts` and can be further inspected.
 
-### Defining a Java fuzz test
+### Java fuzzing
+
+You can write `java_fuzz_tests` through the [Jazzer][jazzer-doc] fuzzing engine. You will need to enable it in your WORKSPACE `rules_fuzzing_dependencies` call:
+
+```python
+rules_fuzzing_dependencies(jazzer = True)
+
+load("@rules_fuzzing//fuzzing:init.bzl", "rules_fuzzing_init")
+
+rules_fuzzing_init()
+
+load("@jazzer//:repositories.bzl", "jazzer_dependencies")
+
+jazzer_dependencies()
+
+load("@jazzer//:init.bzl", "jazzer_init")
+
+jazzer_init()
+```
+
+To use Jazzer, it is convienient to also define a `.bazelrc` configuration, similar to the C++ libFuzzer one above:
+
+```
+# Force the use of Clang for all builds (needed to build Jazzer).
```

Comment:
Would you say that g++ compilability is desirable here? The [most recent Jazzer commit](https://github.com/CodeIntelligenceTesting/jazzer) achieves that and I could update the `repository.bzl` reference. Of course, since `-fsanitize=fuzzer-no-link` is not supported, this would just enable Java-only fuzzing.

Reply:
> Would you say that g++ compilability is desirable here?

I'm not really sure, as I'm not familiar with the typical toolchains used to build native code mixed with Java code. I guess we can consider adding support for it if it ends up breaking some of our users.